src/components/PersonList/PersonList.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+
+interface Props {
+  filteredPeople: Person[];
+  setSelectedPeople: (person: Person) => void;
+}
+
+export const PersonList: React.FC<Props> = React.memo(
+  ({ filteredPeople, setSelectedPeople }) => {
+    return (
+      <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+        <div className="dropdown-content">
+          {filteredPeople.map((person: Person) => (
+            <div
```

Comment:
The best practice is to show `<ul>` and `<li>` tags when you mapping an array with data